doc/doc-site/docs/resources/glossary.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: /glossary
+title: 🍆 Glossary
+hide_title: False
+description: Documentation for emojicoin dot fun
+---
+Degenerate vocabulary you must know
```

Comment:
```suggestion
Degenerate jargon you must know
```

I think jargon is a little more fitting here

Maybe even something like:

~~Degen~~ Defi jargon you must know